### PR TITLE
On 692 privately funded apprentice certificate requests digital solution

### PR DESCRIPTION
**Pull request by gdbarnes**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Application/Handlers/Private/StartPrivateCertificateHandler.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            //    provider = await _assessmentOrgsApiClient.GetProvider(ilr.UkPrn);
+            //}
+            //catch (Exception)
+            //{
+            //    // see whether there are any previous certificates with this ukrpn and a ProviderName....
+            //    var previousProviderName = await _certificateRepository.GetPreviousProviderName(ilr.UkPrn);
+            //    provider = previousProviderName != null
+            //        ? new Provider { ProviderName = previousProviderName }
+            //        : new Provider { ProviderName = "Unknown" };
+            //}
```
Comment (davidgouge) [inline src/SFA.DAS.AssessorService.Application/Handlers/Private/StartPrivateCertificateHandler.cs]: These commented lines can be deleted now. 😁 
And that `"CreateNewCertificate Before Get Standard from API"` log entry.

Comment (jcoxhead) [reply src/SFA.DAS.AssessorService.Application/Handlers/Private/StartPrivateCertificateHandler.cs]: yep:-  I think we are looking at an outdated copy - the comments where removed on friday and I can also see in my local copy they are removed.

Comment (jcoxhead) [reply src/SFA.DAS.AssessorService.Application/Handlers/Private/StartPrivateCertificateHandler.cs]: Hopefully should be ok now.

Comment (jcoxhead) [reply src/SFA.DAS.AssessorService.Application/Handlers/Private/StartPrivateCertificateHandler.cs]: amend bits done

**Pull request closed without merging**